pkgs/tools/security/jadx-bin/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, makeWrapper, jre, unzip, makeDesktopItem }:
+
+let
+  version = "0.9.0";
+  name = "jadx-bin-${version}";
+
+
+  desktopItem = launcher: makeDesktopItem {
+    name = "jadx-gui";
+    exec = "${launcher} %F";
+    comment = "Java Decompiler";
+    desktopName = "jadx GUI";
+    genericName = "Java and Android Decompiler";
+    mimeType = "application/x-java-archive;application/x-java";
+    categories = "Development;Debugger;";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Dex to Java decompiler";
+    homepage    = "https://github.com/skylot/jadx";
+    license     = licenses.asl20;
+    platforms   = platforms.unix;
+    maintainers = [ maintainers.artemist ];
+  };
+
+    src = fetchurl {
```

Comment:
Just ```fetchFromGitHub``` with ```rev = "v${version}"``` maybe?

Reply:
I think that gets you the source, not the binary. I haven't been able to build from source yet.